reporters_db/data/reporters.json

```diff
@@ -10608,6 +10668,26 @@
             }
         }
     ],
+    "L.N.T.S.": [
+        {
+            "cite_type": "treaty",
+            "editions": {
+                "L.N.T.S.": {
+                    "end": "1945-12-31T23:59:59",
+                    "start": "1920-01-01T00:00:00"
+                }
+            },
+            "examples": [
+                "137 L.N.T.S. 11"
+            ],
+            "href": "https://law.resource.org/pub/us/code/blue/IndigoBook.html#T6",
+            "mlz_jurisdiction": [
+                ""
+            ],
```

Comment:
Not sure if this should be empty but it looks wrong to me.

Reply:
Yeah, I don't think anybody uses these, but we should at least populate them until we have decided to remove them. I opened a new issue to see if anybody cares about these over here: #130.

Reply:
I actually use them (I'll put something in the issue too), but these reporters are multi-jurisidictional and its hard to say what the concept of jurisdiction means here (e.g., each treaty is in force to different degrees in different jurisdictions, so presumably a reporter which includes lots of treaties from around the world includes bilateral treaties as well as treaties like the Montreal Protocol, which 198 countries have agreed to). 

For what it's worth, in my use case, I discard the mlz_jurisdiction whenever there is more than one listed.

Reply:
Im okay with leaving this blank for now - but I think we should atleast make an empty array not an array with null in it?  

Thoughts?

Reply:
Made that change!